migrations/4_alter_consent_record_make_valid_to_optional.down.sql

```diff
@@ -0,0 +1,20 @@
+DROP INDEX uniq_record_version;
+
+ALTER TABLE consent_record RENAME TO consent_record_tmp;
+
+CREATE TABLE consent_record (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    patient_consent_id VARCHAR(255) REFERENCES patient_consent(id),
+    valid_from DATE NOT NULL,
+    valid_to DATE NOT NULL,
```

Comment:
Add a default value in 2100 or so.

Then the question is if you're allowed to do an INSERT with null value, does it error or does it use default?
If error, the insert needs to be separated into 2 queries: one with valid_to and one without.

Reply:
Done